subprojects/docs/src/docs/release/notes.md

```diff
@@ -305,6 +305,24 @@ task myTask {
 }
 ```
 
+### Calling `TaskInputs` methods with incorrect parameters
+
+`TaskInputs.file()` used to accept anything that `files()` would, and the same was true for `dir()`. Starting with Gradle 4.3 this behavior is deprecated. Do not do this:
```

Comment:
This is not correct exactly, in the sense that it does not describe what/why it does not accept anymore. May mention the words "single" and "file" or "directory"?

Reply:
Good point.